spec/features/blacklight_customizations/hierarchical_date_facet_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Hierarchical date facet' do
+  before do
+    visit root_path
+    click_link_or_button 'search'
+  end
+
+  it 'includes an expandable date facet' do
+    result_count_display = find('span.page-entries').text
+    expect(result_count_display).to match(/1 - \d+ of \d+/)
+
+    click_link_or_button 'Date'
+
+    find_link('1900-1999').ancestor('li.treeitem').first(:button).click
+    expect(page).to have_content(result_count_display) # have only expanded the facet for 1900s, haven't filtered yet
```

Comment:
try to keep the test from breaking when unrelated fixtures are added, while confirming that simply expanding a tree item doesn't facet on that node immediately.